### ISSUE TITLE
selector-type-no-unknown does not like keyframes with decimal value

### ISSUE THREAD

**Issue by nikolas6** [stylelint/stylelint#1513]
Stylelint 6.7.0 complains when keyframe animation has decimal value like "50.001%".

Sample:
```css
@keyframes loadingAnimation1 {
  0% {
    transform: translate3d(0, 0, 0);
  }
  50% {
    transform: translate3d(100%, 0, 0);
  }
  50.001% {
    transform: translate3d(-100%, 0, 0);
  }
  100% {
    transform: translate3d(0, 0, 0);
  }
}
```

**Comment by jeddy3** [comment]
@nikolas6 I've just pushed up a pending PR for this. Hopefully we can get a patch release out soon.

P.S. Please use the template when creating new issues as it helps us resolve these types of bugs a little quicker.

**Comment by nikolas6** [comment]
Thanks @jeddy3 :)
Noted on the template. My first time reporting issues here.
